action.yml

```diff
@@ -1,55 +1,61 @@
-    default: '_jdk_math'
-  custom_target:   #  Used if need to set non-default custom target
-    description:
-    required: false
-  aqa-testsRepo:
-    description: 'Personal aqa-tests Repo. For example, octocat/aqa-tests:test'
-    required: false
-  aqa-systemtestsRepo:
-    description: 'Personal aqa-systemtests Repo. For example, octocat/aqa-systemtests:test'
-    required: false
-  openj9_repo:
-    description: 'openj9 Repo'
-    required: false
-  tkg_Repo:
-    description: 'Personal TKG Repo. For example, octocat/TKG:test'
-    required: false
-  vendor_testRepos:
-      description: 'Comma-separated list of vendor repositories'
-      required: false
-  vendor_testBranches:
-      description: 'Comma-separated list of vendor branches'
-      required: false
-  vendor_testDirs:
-      description: 'Comma-separated list of vendor directories'
-      required: false
-  vendor_testShas:
-      description: 'Comma-separated list of vendor SHAs'
-      required: false
-runs:
-  using: 'node12'
-  main: 'dist/index.js'
+name: 'Running AQA test'
+description: 'Running AQA test'
+branding:
+  icon: 'check'
+  color: 'purple'
+author: 'Sophia Guo'
+inputs:
+  jdksource:      # upstream, github-hosted, install-jdk, nightly, customized
+    description: 'upstream, github-hosted, install-jdk, nightly, customized'
+    default: 'upstream'
+  customizedSdkUrl:
+    description: 'if jdksource is nightly or customized, this is the url for customized sdk'
+    required: false
+  sdkdir:
+    description: 'if jdksource is nightly or customized, please provide preferred directory to store sdk'
+    required: false
+  version:              # jdk version
+    description: 'testing jdk version (Required when jdksource is github-hosted or install-jdk.)'
+    required: false
+  build_list:           # test category
+    description: 
+    default: 'openjdk'
+  target:              # test
+    description: 
+    default: '_jdk_math'
+  custom_target:   #  Used if need to set non-default custom target
+    description:
+    required: false
+  aqa-testsRepo:
+    description: 'Personal aqa-tests Repo. For example, octocat/aqa-tests:test'
+    required: false
+  aqa-systemtestsRepo:
+    description: 'Personal aqa-systemtests Repo. For example, octocat/aqa-systemtests:test'
+    required: false
+  openj9_repo:
+    description: 'openj9 Repo'
+    required: false
+  tkg_Repo:
+    description: 'Personal TKG Repo. For example, octocat/TKG:test'
+    required: false
+  vendor_testRepos:
+      description: 'Comma-separated list of vendor repositories'
+      required: false
+  vendor_testBranches:
+      description: 'Comma-separated list of vendor branches'
+      required: false
+  vendor_testDirs:
+      description: 'Comma-separated list of vendor directories'
+      required: false
+  vendor_testShas:
+      description: 'Comma-separated list of vendor SHAs'
+      required: false
+  run_parallel:
+      description: 'Run jobs in parallel'

```

Comment:
This parameter is a little bit confusing. Looking at [the workflow example ](https://github.com/j-braley/aqafer/actions/runs/2826544204/workflow) the parameter is used to generate file parallelList.mk not `run jobs in parallel`. I'm thinking maybe we should rename it to make a more clearer?  `run jobs in parallel` is implemented by github action matrix.